docs/resources/new_in_doc.md

```diff
@@ -6,6 +6,26 @@ description: Overview of major recent additions to Ibexa DXP documentation.
 
 This page contains recent highlights and notable changes in [[= product_name =]] documentation.
 
+## June 2024
+
+### Ibexa Engage
+
+- [Learn more about Ibexa Engage](https://doc.ibexa.co/en/latest/ibexa_engage/install_ibexa_engage/)
+
+### Search
+
+- [Configuring Elasticsearch with analyzers for different languages](https://doc.ibexa.co/en/latest/search/search_engines/elastic_search/configure_elastic_search/#add-language-specific-analysers)
+- [ContentName search criterion](https://doc.ibexa.co/en/master/search/criteria_reference/contentname_criterion/)
+
+### Dashboard
+
+- [Customizing the content type for Dashboard container](https://doc.ibexa.co/en/latest/administration/dashboard/configure_default_dashboard/#container-content-type-identifier)
+
+### Infrastructure and maintenance
+
+- [Updated Ibexa Cloud domain to ibexa.cloud](https://doc.ibexa.co/en/latest/getting_started/install_on_ibexa_cloud/#4-push-the-project)
+- [v4.6.3 to v4.6.4 update instructions](https://doc.ibexa.co/en/master/update_and_migration/from_4.5/update_from_4.5/#v464)
```

Comment:
Reading this, I feel like we should have a way to ensure the reader that we aren't out-dated, that there is just nothing to do to upgrade from 4.6.4 to 4.6.8.

Reply:
Shouldn't it redirect to master, not latest in the link?

Reply:
> Reading this, I feel like we should have a way to ensure the reader that we aren't out-dated, that there is just nothing to do to upgrade from 4.6.4 to 4.6.8.

I see what you mean, so we could mention all the releases in the update doc - saying that there is nothing to do? Sounds clearer to me, good idea.